### PR TITLE
2299 - device creation step 2

### PR DESCRIPTION
**Pull request by luisfelipefurlan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
added security step on device creation

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#2299

* **Other information**:

### CODE REVIEW

File: src/js/views/createDevice/steps/SecurityStep/SecurityTable.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                >
+                  <TableCell padding='checkbox'>
+                    <Radio
+                      color='primary'
+                      checked={isSelected}
+                      onChange={() => handleCertificateSelection(cert)}
+                    />
+                  </TableCell>
+
+                  <TableCell className={classes.clickableCell}>Nome do Certificado</TableCell>
```
Comment (LuanEdCosta) [inline src/js/views/createDevice/steps/SecurityStep/SecurityTable.jsx:96]: Maybe you can use the certificate attributes here, example: `cert.label`, `cert.created`, `cert.expiration`. You can also use moment to format dates.

File: src/js/common/components/DataTable/DataTableHead.jsx [hunk 2]
```diff
@@ -25,13 +26,15 @@ const DataTableHead = ({
...
       <TableRow>
-        <TableCell>
-          <Checkbox
-            color='primary'
-            onChange={onSelectAllClick}
-            checked={rowCount > 0 && numSelected === rowCount}
-            indeterminate={numSelected > 0 && numSelected < rowCount}
-          />
+        <TableCell padding='checkbox'>
+          {disableCheckbox ? undefined : (
```
Comment (LuanEdCosta) [inline src/js/common/components/DataTable/DataTableHead.jsx:30]: I think that you can use the prop `disableCheckbox` to hide the `<TableCell>` on line 29 too. You can use the `startExtraCells` prop to add an extra `<TableCell>` in this component: `src/js/views/createDevice/steps/SecurityStep/SecurityTable.jsx`.

**Review by LuanEdCosta — CHANGES_REQUESTED** [review]
Very good job :+1: 
---

**Review by cfrancisco — DISMISSED** [review]

**Review by LuanEdCosta — APPROVED** [review]
:rocket:

**Pull request merged**